src/Canvas.cc

```diff
@@ -914,8 +914,6 @@ Canvas::resurface(Local<Object> canvas) {
   Nan::HandleScope scope;
   Local<Value> context;
 
-  backend()->recreateSurface();
```

Comment:
Would you mind explaining this change? I'm not to familiar with this part of the code 😅

Reply:
I don't remember the exact details, but reviewing the PR, seems to me like a bug, because `recreateSurface()` it's not being called anywhere else...

Reply:
Maybe best to add it back?

Reply:
Maybe, or test it first as is, to be sure if it crash or fails.